src/serve/ownerSizeTotalWorker.test.ts

```diff
@@ -0,0 +1,336 @@
+'use strict'
+
+import { sbp, assert, path, okturtlesData, okturtlesEvents, okturtlesEventQueue } from '../deps.ts'
+import { createCID } from '../deps.ts'
+import { appendToIndexFactory, initDB, updateSize as updateSize_ } from './database.ts'
+
+// Type for Deno test context
+type TestContext = {
+  step(name: string, fn: () => void | Promise<void>): Promise<boolean>
+}
```

Comment:
There shouldn't be a need for this. It should either be inferred or imported from Deno.